src/EA.Weee.RequestHandlers/AatfEvidence/SaveEvidenceNoteRequestBase.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace EA.Weee.RequestHandlers.AatfEvidence
+{
+    using System;
+    using Core.Helpers;
+    using Domain.AatfReturn;
+
+    public abstract class SaveEvidenceNoteRequestBase
+    {
+        public void AatfIsValidToSave(Aatf aatf, DateTime systemDateTime)
+        {
+            if (aatf.AatfStatus != AatfStatus.Approved || !WindowHelper.IsDateInComplianceYear(aatf.ComplianceYear, systemDateTime) 
+                                                       || (aatf.ApprovalDate.HasValue && aatf.ApprovalDate.Value.Date > systemDateTime.Date))
+            {
+                throw new InvalidOperationException(
+                    $"Aatf with Id {aatf.AatfId} has invalid status to save");
```

Comment:
This actually doesnt get bubbled to the front end. It shouldnt ever happen as the should be prevented from the front end. Its a last check in case user does something horrid.

Reply:
okay